deploy/olm-catalog/konveyor-operator/latest/konveyor-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -173,7 +173,6 @@ metadata:
       ]
     certified: "false"
     support: Red Hat
-    operatorframework.io/cluster-monitoring: "true"
```

Comment:
My impression from submitting PRs to community operators is that we should not have this upstream. It will instead be added by the downstream-prep script now.